### PR TITLE
feature/46_PUBLISHED_UNPUBLISED_STATUS_IN_LISTING_INDEX

### PR DESCRIPTION
**Pull request by LupusNocti**
- Modified the listings view to filter the published listings

### CODE REVIEW

File: resources/views/listings/index.blade.php [hunk 1]
```diff
@@ -23,57 +27,37 @@
...
-                        <!-- User's Name and Date/Time -->
-                        <div class="ms-3">
-                            @if ($listing->user)
-                            <h5 class="card-title" id="listing-user-name">{{ $listing->user->name }}</h5>
-                            @endif
-                            <p class="card-text" id="listing-user-position">Software Engineer</p>
-                            <p class="card-text" id="listing-datetime">Posted on: {{ $listing->created_at->format('F d, Y H:i:s') }}</p>
-                        </div>
-                    </div>
+                @if($listing->status === ListingAction::PUBLISH)
```
Comment (ChronoDevs) [inline resources/views/listings/index.blade.php]: please declare whole path App\Enums\ListingAction::PUBLISH

File: resources/views/listings/index.blade.php [hunk 2]
```diff
@@ -3,6 +3,10 @@
 @section('title', 'Listings')
 @section('css', 'listings_index.css')
 
+@php
```
Comment (ChronoDevs) [inline resources/views/listings/index.blade.php]: please remove

File: app/Http/Controllers/ListingController.php [hunk 3]
```diff
@@ -6,6 +6,7 @@
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Gate;
 use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Config;
```
Comment (ChronoDevs) [inline app/Http/Controllers/ListingController.php]: Please confirm, where did you use this?

Comment (LupusNocti) [reply app/Http/Controllers/ListingController.php]: Hi sir @ChronoDevs, Good morning. That is for the pagination located in the config/const. :bow:

Comment (ChronoDevs) [reply app/Http/Controllers/ListingController.php]: I think it's working without redeclaration

Comment (LupusNocti) [reply app/Http/Controllers/ListingController.php]: I see. Okay sir, I will remove it :blush:

**Pull request merged**